src/stylesheets/components/_accordions.scss

```diff
@@ -90,3 +90,7 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-bottom: 0;
   }
 }
+
+.usa-accordion-content-collapsed {
```

Comment:
@joshbruce you're right, there may not be any benefit to the `visibility: hidden` method, that I could think of, but it might require more research. Anyhow, I tried a simple, classless method here: #1196. I'd like to hear what others think.